src/v2/Apps/Auctions/AuctionsApp.tsx

```diff
@@ -52,21 +61,28 @@ const AuctionsApp: React.FC<AuctionsAppProps> = props => {
         </Column>
       </GridColumns>
 
-      {user && (
-        <>
-          <Spacer py={2} />
-          <Join separator={<Spacer my={[2, 6]} />}>
-            <Box>
-              {/* @ts-expect-error STRICT_NULL_CHECK */}
-              <MyBidsFragmentContainer me={viewer.me} />
-            </Box>
+      <Spacer mt={4} />
+      <Tabs>
+        {user && (
+          <>
+            <Tab name="Works For You">
+              <Spacer py={2} />
+              <Box>
+                {/* @ts-expect-error STRICT_NULL_CHECK */}
```

Comment:
If we can fix this type error before merging 🙏

Reply:
Resolved.